### PR TITLE
Fix refferential file/json upload validations

### PR DESCRIPTION
**Pull request by evgshk**

### CODE REVIEW

File: src/MarginTrading.AssetService/Controllers/AssetsController.cs [hunk 1]
```diff
@@ -76,6 +76,18 @@ public async Task<List<string>> GetUsedIsins()
...
         }
+        
+        /// <summary>
+        /// Returns used product names
+        /// </summary>
+        [HttpGet]
+        [Route("used-names")]
+        public async Task<List<string>> GetUsedNames()
+        {
+            var data = await _assetsRepository.GetUsedNamesAsync();
```
Comment (tarurar) [inline src/MarginTrading.AssetService/Controllers/AssetsController.cs:87]: Will it work fine if we have 1000, 10000 different product names?

Comment (evgshk) [reply src/MarginTrading.AssetService/Controllers/AssetsController.cs:87]: Well, it's been implemented right as ISIN validation has been done before.

 If we decide it doesn't fulfill requirements there is another way to do a uniqueness check. We pass new names as an argument and check if there are no duplicates on the AssetService side.

Comment (evgshk) [reply src/MarginTrading.AssetService/Controllers/AssetsController.cs]: @tarurar implementation has been updated

File: src/MarginTrading.AssetService.Contracts/IAssetsApi.cs [hunk 2]
```diff
@@ -27,12 +27,18 @@ public interface IAssetsApi
         /// </summary>
         [Get("/api/assets/used-isins")]
         Task<List<string>> GetUsedIsins();
```
Comment (tarurar) [inline src/MarginTrading.AssetService.Contracts/IAssetsApi.cs:29]: Do we still need this one once we have new endpoint `GetDuplicatedIsins`?

Comment (evgshk) [reply src/MarginTrading.AssetService.Contracts/IAssetsApi.cs]: Due to performance issues we remove this method. It's now required to use `GetDuplicatedIsins()` for uniqueness check.

**Review by tarurar — APPROVED** [review]

**Comment by evgshk** [comment]
should be declined in future

**Pull request closed without merging**